sdk-core/transactions-factories/token_management_transactions_factory.md

```diff
@@ -0,0 +1,174 @@
+## TokenManagementTransactionsFactory
+
+A class that provides methods for creating transactions for token management operations.
+
+```
+class TokenManagementTransactionsFactory:
+    createIssueFungibleTransaction({
+        sender: IAddress;
+        tokenName: string;
+        tokenTicker: string;
+        canFreeze: boolean;
+        canWipe: boolean;
+        canPause: boolean;
+        canTransferNFTCreateRole: boolean;
+        canChangeOwner: boolean;
+        canUpgrade: boolean;
+        canAddSpecialRoles: boolean;
+
+        // Optionals:
```

Comment:
irrelevant fields for these optionals. This factory should have functions that will be able to correctly create the transaction data, and provide the minimum gas limit for the transaction to run and the value required.

Reply:
Correct. Irrelevant optionals (value) should be removed. However, I would keep nonce and gas limit (opinion).

Nonce - because the developer might choose to provide the account nonce when calling the function of the factory, if this nonce is already available. If not, the developer can call `setNonce` anyway, at a later time.

Gas limit - to ameliorate the effect of eventual bugs in the SDKs - for example, incorrectly computing gas limit for altering NFT attributes. In case of such a bug, the developer could simply provide an explicit, increased gas limit - until a proper fix is released in the SDKs.

Reply:
`Value` removed. The other kept, would be removed upon re-discussing.

Reply:
As agreed in the call, these function should return a new type of struct: TransactionIntent. Optionals will no longer be part of the definitions